recipes/adstex/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "adstex" %}
+{% set version = "0.3.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/adstex-{{ version }}.tar.gz
+  sha256: 9a192841625e4f79480bf430c5a9631060f64dee4ece018f4a992cfe7d605e49
+
+build:
+  number: 0
+  entry_points:
+    - adstex=adstex:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - ads >=0.12.3
+    - bibtexparser >=0.6.2
+    - future >=0.12.0  # [py<30]
```

Comment:
Do you really need Python 2.7 support? Right now conda-forge isn't even building Python 2.7 packages by default.

Reply:
No. Python 3+ is fine. I actually did that in 48fdb2a3 but the linter said that "Non noarch packages should have python requirement without any version constraints." So I then removed the Python version requirement.  

Can you suggest what may be a good way to edit the recipe? Thank you!.

Reply:
Well for this line, you should be able to completely eliminate the dependency on `future`.

Reply:
Thank you for the tips! I've implemented your suggestions.